guides/WRITING-DOCUMENTATION.md

```diff
@@ -10,11 +10,9 @@ For a component named "Foo", you would have the following:
 
 ```
 └── components
-    └── Foo                     A directory holding all relevant component files
-        ├── Foo.scss            Styling and KSS comment blocks
-        ├── Foo.jsx             React component
-        ├── Foo.test.jsx        React component tests
-        └── Foo.example.jsx     React component used to render an example
+    └── Foo                     A directory holding all relevant code
```

Comment:
I feel like this description isn’t super helpful. It’s a directory holding documentation content and examples, we don’t want to confuse it with the source code

Reply:
Another though, we should probably show how the documentation files are separated from the source code. Maybe have a separate section that shows the file structure of ‘docs/src/pages/components’?